packages/test-helpers/src/bin/run.js

```diff
@@ -1,8 +1,9 @@
 #!/usr/bin/env node
 
+import githubCommentManager from "./github-comment-manager";
+
 const program = require("commander");
```

Comment:
should be able to import the requires now that we are transpiling

last thing promise!